src/index.html

```diff
@@ -16,17 +16,25 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
+    <div class="input input__icon input__icon--big">
+      <input
+        class="input__big"
+        type="text"
+        data-qa="keypress big"
+        placeholder="Try “Los Angeles“"
+      />
+      <!-- <span class="input__icon"></span> -->
+    </div>
```

Comment:
The class names `input__icon` and `input__icon--big` suggest that they might be related to an icon inside the input, but there is no icon element present. Ensure class names represent the meaning of the content, not the styles or tag names.